CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.12)
 project(torchvision)
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Is this necessary? My understanding is that TorchVision aligns with PyTorch
s approach and sets `CMAKE_CXX_STANDARD` to 14.

Reply:
This is necessary to fix Windows compilation errors, where CUDA-11.x (which supports C++17 standard) could not be compiled using C++14 standard.

Reply:
Thanks for the clarification. Makes sense. So I guess PyTorch Core will be moving as well to C++17, right?